post-install.yml

```diff
@@ -10,15 +10,9 @@
 
   tasks:
 
-    - name: Install islandora_defaults module
-      command: "{{ drush_path }} --root {{ drupal_core_path }} -y en islandora_defaults"
-
-    - name: Import feature
-      command: "{{ drush_path }} --root {{ drupal_core_path }} -y fim --bundle=islandora islandora_defaults"
```

Comment:
Why are you not enabling islandora_defaults and importing it anymore? Did it move somewhere else? When I ran a fresh build, I lost a lot of the settings like having media use on the media types.

Reply:
I think it is being enabled (as a module) @dannylamb added it to the `drupal_enabled_modules` ansible variable, but we probably still need this line to import the feature because it is not imported by default.

Reply:
Indeed, I manually ran drush en and it was already enabled, then I ran the drush fim and it fixed the missing things 👍  (and then a FITS media object was made! )